lib-src/libsoxr/src/pffft-wrap.c

```diff
@@ -40,7 +40,7 @@ static void pffft_zconvolve(PFFFT_Setup *s, const float *a, const float *b, floa
 
   float ar, ai, br, bi;
 
-#ifdef __arm__
+#if defined(__arm__) || defined (_M_ARM64)
```

Comment:
Do we need __aarch64__ here? Probably not, as the code builds for Apple Silicon and RPi

Reply:
Yeah, I figured, that I will only add the ARM64 flags and not __aarch64__. Did notice a few places, where they could be, but I was not very sure.

Reply:
In any case, it's only a prefetch :-)